watch-and-repack.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Relative paths to the web repositories
+PENUMBRA_ZONE_WEB_PATH="../../web"
+PRAX_REPO_PATH="."
+
+# Ensure 'pack-public.sh' is executable
+chmod +x "$PENUMBRA_ZONE_WEB_PATH/pack-public.sh"
+
+# Remove any existing tarballs from packages in penumbra-zone/web 
+cleanup_tarballs() {
+  find "$PENUMBRA_ZONE_WEB_PATH/packages" -name "*.tgz" -type f -delete
+}
+
+# Repack the packages in penumbra-zone/web
+repack() {
+  (cd "$PENUMBRA_ZONE_WEB_PATH" && ./pack-public.sh)
```

Comment:
should probably allow other repos to control their own pack behavior, and place output in an external target directory.